controls/oracle19c-3.2.rb

```diff
@@ -41,7 +41,7 @@
      WHERE PROFILE='DEFAULT'
      AND RESOURCE_NAME='PASSWORD_LOCK_TIME'
      AND CON_ID = P.CON_ID),
-     'UNLIMITED','9999',P.LIMIT)) < 1
+     'UNLIMITED','9999',P.LIMIT)) < #{input('password_lock_time')}
```

Comment:
@ejaronne I'm pretty sure that the `1` on line 3 of the query needs to be left alone:
```
SELECT P.PROFILE, P.RESOURCE_NAME, P.LIMIT,
DECODE (P.CON_ID,0,(SELECT NAME FROM V$DATABASE),	    
1,(SELECT NAME FROM V$DATABASE), ...
```
Please let me know if you think otherwise

Reply:
Since [DECODE](https://docs.oracle.com/en/database/oracle/oracle-database/12.2/sqlrf/DECODE.html) looks for search results, here, it's looking for value `1` under `P.CON_ID`. So this has nothing to do with `PASSWORD_LOCK_TIME`